handbook/engineering/roles.md

```diff
@@ -26,30 +27,56 @@ Some teams with more experienced members may need less guidance in certain areas
 ### Responsibilities
 
 - Directly contribute through code, RFCs or PR reviews for a minimum of roughly 50% of their time.
-- Collaborate with [product](../product/index.md) to maintain a [prioritized roadmap](../../direction/index.md) for the work owned by their team as well as quarterly [OKRs](../../company/okrs/index.md)
-- Plan and commit to a set of work that will be accomplished by the team for each [monthly release](releases/index.md) and capture those commitments in a [tracking issue](tracking_issues.md) on GitHub.
+- Ensure that the team has a roadmap that documents how it is going to achieve its [OKRs](../../company/okrs/index.md).
+- Ensure the the team's [tracking issue](tracking_issues.md) captures the work that each teammate has committed to for each [monthly release](releases/index.md).
+- Post a weekly update by EOD Monday on the tracking issue that answers the following questions:
+  1. What work did the team do last week?
+    - This is a summary not details. The details should exist in each teammate's individual update and shouldn't be repeated.
+  1. How does that work connect back to the team’s OKRs and roadmap?
+  1. For each OKR and roadmap item, is the team on track? The answer should be of the form:
```

Comment:
To be clear, for this, should the format be a list of each of our team's items from https://about.sourcegraph.com/company/okrs/2020_q2 and https://docs.google.com/document/d/1cBsE9801DcBF9chZyMnxRdolqM_1c2pPyGQz15QAvYI/edit#heading=h.5nwl5fv52ess, and an accompanied status update?

Reply:
Yes on OKRs, but rather than the entirety of the roadmap, it's the bigger projects that are in progress.

For example the search team this would be:
- RFC 159
- Indexing non-master branches
- AND/OR queries/hierarchical search
- Zoekt performance

Reply:
I moved the docs to the tracking issue page and updated the wording. PTAL and suggest any edits that you think would make it more clear.